docs/docs/getting-started/install.md

```diff
@@ -1,26 +1,26 @@
 ---
-title: Install
-layout: article
-category: install
-permalink: /install
+title: Installation
+sidebar_position: 2
 ---
 
+:::note
+
+This guide is assuming you are using Ubuntu 18.04 LTS for installation, which comes with `ufw` and `netplan`.
+
+:::
+
 ## Minimum System Requirements
 
 - One of the following Linux distributions
   - Ubuntu [current LTS](https://wiki.ubuntu.com/Releases)
   - Debian [current stable](https://www.debian.org/releases/)
-- 1 Ghz Processor
+- A modern, multi-core processor (minimum of two cores for virtual machines)
```

Comment:
Both previous and new cpu requirements are really vague, but 1Ghz feel more like a minimum requirement as "A modern, multi-core processor", people will always be able to scale up if speed is required.